mithril-common/src/messages/certificate_list.rs

```diff
@@ -160,25 +153,54 @@ impl Debug for CertificateListItemMessage {
 
 #[cfg(test)]
 mod tests {
+    use crate::entities::CardanoDbBeacon;
+
     use super::*;
 
-    type CertificateListMessagePreviousVersion = Vec<CertificateListItemMessagePreviousVersion>;
+    const ACTUAL_JSON: &str = r#"[{
+            "hash": "hash",
+            "previous_hash": "previous_hash",
+            "epoch": 10,
+            "signed_entity_type": { "MithrilStakeDistribution": 10 },
+            "metadata": {
+                "network": "testnet",
+                "version": "0.1.0",
+                "parameters": {
+                    "k": 1000,
+                    "m": 100,
+                    "phi_f": 0.123
+                },
+                "initiated_at": "2024-02-12T13:11:47Z",
+                "sealed_at": "2024-02-12T13:12:57Z",
+                "total_signers": 2
+            },
+            "protocol_message": {
+                "message_parts": {
+                    "snapshot_digest": "snapshot-digest-123",
+                    "next_aggregate_verification_key": "next-avk-123"
+                }
+            },
+            "signed_message": "signed_message",
+            "aggregate_verification_key": "aggregate_verification_key"
+        }]"#;
+
+    type CertificateListMessageUntilV0_1_32 = Vec<CertificateListItemMessageUntilV0_1_32>;
 
     #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-    struct CertificateListItemMessagePreviousVersion {
+    pub struct CertificateListItemMessageUntilV0_1_32 {
```

Comment:
This struct doesn't need to be public.
```suggestion
    struct CertificateListItemMessageUntilV0_1_32 {
```